elevenlabs_mcp/server.py

```
@@ -1215,6 +1215,72 @@ def create_composition_plan(
+    Args:
+        tool_id: The ID of the tool to delete
+
+    Returns:
+        TextContent with a simple success message when deleted
+    """
+    if tool_id == "":
+        make_error("Tool ID is required.")
+    try:
+        client.conversational_ai.tools.delete(tool_id=tool_id)
+        return TextContent(type="text", text=f"Tool deleted successfully: {tool_id}")
+    except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)